File: C++/1142.cpp
#include <iostream>

using namespace std;

int main() {
    
    int n, i, a = 1, b = 2, c = 3;
    cin >> n;
    
    for(i = 0; i < n; i++) {
        cout << a << " " << b << " " << c << " PUM" << endl;
        a+=4;
        b+=4;
        c+=4;
    }
    
    return 0;
}

File: javascript/1142.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var valor = parseInt(input);
var [a,b,c] = [1,2,3];

for (let i = 1; i <= valor; i++) {
    console.log(`${a} ${b} ${c} PUM`);
    a += 4;
    b += 4;
    c += 4;
}